homeassistant/components/vallox/services.yaml

```
@@ -27,3 +27,23 @@ set_profile_fan_speed_boost:
           min: 0
           max: 100
           unit_of_measurement: "%"
+
+set_profile:
+  fields:
+    profile:
+      required: true
+      selector:
+        select:
+          options:
+            - "Home"
+            - "Away"
+            - "Boost"
+            - "Fireplace"
+            - "Extra"
+    duration:
+      required: false
+      selector:
+        number:
+          min: 1
+          max: 65535
+          unit_of_measurement: "min"
```

Maybe it's more clear to avoid an abbreviation here:

```suggestion
          unit_of_measurement: "minutes"
```

---

does it get a label from some translation or is it just a literal that's shown on the service page

---

apparently latter, looking at other service definitions